api.ts

```diff
@@ -77,7 +77,7 @@ export class StatusCodeError extends Error {
   options: FetchRequest;
   response: StatusCodeErrorResponse;
   constructor(statusCode: number, body: any, options: FetchRequest, response: StatusCodeErrorResponse) {
-    super(`${statusCode} - ${body}`);
+    super(`Fetch failed with status code: ${statusCode} - ${JSON.stringify(body, null, 4)}`);
```

Comment:
I think `util.inspect()` is preferable. Expands a full object into a string, without transforming it.

Reply:
util is a node library, while this is used in thunk, non-browserified. I don't want to inject `util` just for this.

Reply:
Ah ok, makes sense, too bad.